src/js/themes/base.js

```diff
@@ -945,7 +947,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       border: {
         radius: '4px',
-        // intelligentRounding: undefined,
+        intelligentRounding: true,
```

Comment:
```suggestion
        // intelligentRounding: undefined,
```
Don't enable by default

Reply:
oh this makes sense why that was there